### PR TITLE
fix api docs path

### PR DESCRIPTION
**Pull request by ryelo**
adding a reload to the api-docs so that it directs to `docs/api`. If this isn't right, someone can feel free to commit to this branch as I'm going to be out

### CODE REVIEW

**Comment by kruai** [comment]
@ryelo can you please remove the 2 extra `config.yml` files?

File: beta/config/main.yml [hunk 1]
```diff
@@ -0,0 +1,450 @@
+ansible:
```
Comment (karelhala) [inline beta/config/main.yml]: I don't think you should push this file.

File: config/main.yml [hunk 2]
```diff
@@ -0,0 +1,448 @@
+ansible:
```
Comment (karelhala) [inline config/main.yml]: Or this one.

File: beta/config/main.yml (hunk 1 above)
Comment (ryelo) [reply beta/config/main.yml]: whoops. I thought I had added these to my gitignore

**Comment by ryelo** [comment]
@kruai @karelhala So now BU wants to remove product docs because some criteria changed. The path for api docs is still going to be docs/api, but I'm removing the product docs from the bundle for now.

**Review by kruai — APPROVED** [review]

**Review by karelhala — APPROVED** [review]

**Pull request merged**